docs/buildah.md

```diff
@@ -26,7 +26,24 @@ Print debugging information
 
 **--default-mounts-file**
 
-Path to default mounts file (default path: "/usr/share/containers/mounts.conf")
+Path to default mounts file (default path: "/usr/share/containers/mounts.conf") Alternate path used `/etc/containers/mounts.conf` if user created it.
```

Comment:
Suggest: ") Alternate path used ..." to ").  The alternate path is `/etc/containers/mounts.conf` if created by the user. "